packages/rscontext/rscontext/rs_context.py

```diff
@@ -108,6 +110,9 @@ def rs_context(huc, existing_veg, historic_veg, ownership, ecoregions, output_fo
     nhd, db_path, huc_name = clean_nhd_data(huc, nhd_download_folder, nhd_unzip_folder, os.path.join(output_folder, 'hydrology'), cfg.OUTPUT_EPSG, False)
     project.add_metadata({'Watershed': huc_name})
 
+    precip = mean_area_precip(nhd['WBDHU{}'.format(len(huc))], prism)
```

Comment:
can we set up the geopackage as a project layer the same way the others work?

https://github.com/Riverscapes/riverscapes-tools/blob/master/packages/rscontext/rscontext/rs_context.py#L40-L51

This is going to make it easier to write to the project.rs.xml file.

Reply:
Added as a new layer type. I'm not sure what else needs to be done with this for now.